core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -46,33 +47,14 @@ public ConnectorMaterializedViewDefinition(
             @JsonProperty("comment") Optional<String> comment,
             @JsonProperty("owner") String owner,
             @JsonProperty("properties") Map<String, Object> properties)
-    {
-        this(originalSql, requireNonNull(storageTable, "storageTable is null").orElse(null), catalog, schema, columns, comment, Optional.of(owner), properties);
-    }
-
-    /*
```

Comment:
Please make `ConnectorMaterializedViewDefinition` cleanup separate from decoupling in Iceberg